pygmt/tests/test_xyz2grd.py

```diff
@@ -19,40 +19,52 @@ def fixture_ship_data():
     return load_sample_bathymetry()
 
 
-def test_xyz2grd_input_file():
```

Comment:
Why do you remove the `test_xyz2grd_input_file` test?

Reply:
I wanted to standardize it with the other modules that import remote files using a dedicated module (in this case, `load_sample_bathymetry()`). I can re-add it, but I figured it was a redundant test, as it is testing the GMT function to call the remote file rather than the PyGMT function to convert the information to a pandas dataframe or numpy array.